jbpm-wb-process-runtime/jbpm-wb-process-runtime-client/src/main/java/org/jbpm/workbench/pr/client/editors/instance/list/ProcessInstanceSummaryErrorPopup.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2017 Red Hat, Inc. and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.jbpm.workbench.pr.client.editors.instance.list;
+
+import javax.annotation.PostConstruct;
+import javax.enterprise.context.Dependent;
+import javax.inject.Inject;
+
+import com.google.gwt.core.shared.GWT;
+import com.google.gwt.event.dom.client.ClickEvent;
+import com.google.gwt.event.dom.client.ClickHandler;
+import com.google.gwt.user.client.ui.Composite;
+import com.google.gwt.user.client.ui.PopupPanel;
+import org.gwtbootstrap3.client.ui.Anchor;
+import org.gwtbootstrap3.client.ui.html.Span;
+import org.jboss.errai.ui.shared.api.annotations.DataField;
+import org.jboss.errai.ui.shared.api.annotations.Templated;
+import org.jbpm.workbench.pr.client.resources.i18n.Constants;
+
+@Dependent
+@Templated(value = "ProcessInstanceSummaryErrorPopup.html")
+public class ProcessInstanceSummaryErrorPopup extends Composite implements ClickHandler{
+    
+    @Inject
+    @DataField("displayText")
+    public Span displayText;
+    
+    @Inject
+    @DataField("viewLink")
+    public Anchor viewLink;
+    
+    private static Constants constants = Constants.INSTANCE;
+    
+    private long processInstanceId = -1;
+    
+    PopupPanel popupPanel = new PopupPanel(true);
+
+    public ProcessInstanceSummaryErrorPopup(){
+    }
+    
+    @PostConstruct
+    public void init(){
+        viewLink.addClickHandler(this);
+    }
+    
+    public void setData(long pid, int errCount){
+        displayText.setText(constants.ErrorCountNumber(errCount));
+        viewLink.setText(constants.ErrorCountViewLink());
+        processInstanceId = pid;
+    }
+    
+    public void showAt(int x, int y){
+        if(!popupPanel.isShowing()){
+            popupPanel.setWidget(this);
+            popupPanel.setPopupPosition(x + 10, y - 28);
+            popupPanel.show();
+        }
+    }
+    
+    public void hide(){
+        popupPanel.hide();
+    }
+    
+    @Override
+    public void onClick(ClickEvent event) {
+        //TODO: go to error view, filtering by process instance ID (depends on JBPM-5950)
+        GWT.log("Clicked on link for PID: " + processInstanceId); //TODO: remove log trace
```

Comment:
Here I understand it should call `placeManager.goTo( "Error Perspective" );` or similar